### PR TITLE
OP-265 do not depend on all values being the same with constructor method

### PR DESCRIPTION
**Pull request by dbmalkovsky**

### CODE REVIEW

File: src/main/java/org/isf/accounting/manager/BillBrowserManager.java [hunk 1]
```diff
@@ -69,9 +69,14 @@ protected void validateBill(Bill bill,
         List<OHExceptionMessage> errors = new ArrayList<OHExceptionMessage>();
         
         GregorianCalendar today = new GregorianCalendar();
```
Comment (mwithi) [inline src/main/java/org/isf/accounting/manager/BillBrowserManager.java:71]: Fix indentation please

**Review by mwithi — APPROVED** [review]
Approved.

File: src/main/java/org/isf/accounting/manager/BillBrowserManager.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/main/java/org/isf/accounting/manager/BillBrowserManager.java:71]: Done; everything is inline now.

Comment (mwithi) [reply src/main/java/org/isf/accounting/manager/BillBrowserManager.java:71]: Thanks a lot!!!

**Pull request merged**